src/CMakeLists.txt

```diff
@@ -68,30 +68,29 @@ if(CMAKE_BUILD_TYPE MATCHES Debug)
   log_option_disabled("DEBUG LOG")
 endif(CMAKE_BUILD_TYPE MATCHES Debug)
 
-
 # *****************************************************************************
 # Packages / Libs
 # *****************************************************************************
 if (MSVC)
-  find_package(Boost 1.53.0 COMPONENTS system filesystem iostreams date_time REQUIRED)
+  find_package(Asio REQUIRED)
   find_package(cryptopp CONFIG REQUIRED)
   set(CRYPTOPP_LIBRARIES "cryptopp-static")
   find_package(CURL REQUIRED)
   find_package(jsoncpp REQUIRED)
   find_package(LuaJIT REQUIRED)
   find_package(MySQL REQUIRED)
   find_package(PugiXML REQUIRED)
-  find_package(spdlog CONFIG REQUIRED)
+  find_package(spdlog REQUIRED)
   find_package(Threads REQUIRED)
   find_package(libzippp CONFIG REQUIRED)
 else()
-  find_package(Boost REQUIRED COMPONENTS system filesystem iostreams date_time)
+  find_package(Asio REQUIRED)
   find_package(cryptopp CONFIG REQUIRED)
   find_package(CURL CONFIG REQUIRED)
   find_package(jsoncpp CONFIG REQUIRED)
   find_package(LuaJIT REQUIRED)
   find_package(pugixml CONFIG REQUIRED)
-  find_package(spdlog CONFIG REQUIRED)
+  find_package(spdlog REQUIRED)
   find_package(Threads REQUIRED)
   find_package(unofficial-libmariadb CONFIG REQUIRED)
   find_package(libzippp CONFIG REQUIRED)
```

Comment:
Create if/else only for the packages that changes, for example 'find_package(spdlog CONFIG REQUIRED)' is equal for both, so move to before if, and remove from the if/else, etc